src/main/scala/nl.knaw.dans.easy.dd2d/fieldbuilders/AbstractFieldBuilder.scala

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (C) 2020 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.dd2d.fieldbuilders
+
+import nl.knaw.dans.lib.dataverse.model.dataset.MetadataField
+
+abstract class AbstractFieldBuilder {
+
```

Comment:
Since all extending classes have the addValue method maybe add: def addValue(v: AnyRef): Unit

Reply:
Good point. It is a trade-off between type safety and avoid repetitive code. I went with the former here. I have made a short-lived attempt to use a type parameter, but that conflicted with the code in `nl.knaw.dans.easy.dd2d.DepositToDataverseMapper`.